Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorDrawEffect.py

```diff
@@ -16,6 +17,7 @@ class SegmentEditorDrawEffect(AbstractScriptedSegmentEditorLabelEffect):
 
     def __init__(self, scriptedEffect):
         scriptedEffect.name = 'Draw'
```

Comment:
Add a comment that this should not be translated (to prevent accidental translation by other developers in the future):
```suggestion
        scriptedEffect.name = 'Draw'  # no tr (effect name must not be translated because modules find effects by this name)
```